### PR TITLE
Improve HACKING.md

### PR DESCRIPTION
**Pull request by lmuntaner**
Fix a command of the HACKING.md

<hr/><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

**Comment by lmuntaner** [comment]
@frederikrothenberger @nmattia please review

File: HACKING.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ See `dfx canister call --help` and [the documentation](https://sdk.dfinity.org/d
 The `dfx` executable can proxy queries to the canister. To view it, run the following and open the resulting link in your browser:
 
 ```bash
-echo "http://localhost:4943?canisterId=$(dfx canister id internet_identity)"
```
Comment (frederikrothenberger) [inline HACKING.md:56]: Do you still have a global network config? I think it should work fine with the default port 4943. The switch to the subdomain based routing is good though. 👍

Comment (frederikrothenberger) [reply HACKING.md:56]: Well, I suppose that is because port `8080` is configured in `/Users/llorencmuntaner/.config/dfx/networks.json`

Comment (lmuntaner) [reply HACKING.md:56]: Yes, but I didn't touch that file, I assumed everybody has the same unless they change it. Or is it me the only one using that port?

Comment (nmattia) [reply HACKING.md:56]: @przydatek had that issues as well, worth figuring out how 8080 got there but as Frederik said 4943 is the default so let's use that

Comment (lmuntaner) [reply HACKING.md:56]: 8080 is needed for `dfx nns install`.

https://dfinity.slack.com/archives/CGA566TPV/p1708008254336109?thread_ts=1708007865.314369&cid=CGA566TPV

**Review by nmattia — APPROVED** [review]
Thanks! Might want to point out that this doesn't work in e.g. Safari though

File: HACKING.md (hunk 1 above)
Comment (lmuntaner) [reply HACKING.md:56]: Done!

**Comment by peterpeterparker** [comment]
Wait, most people do not use `dfx nns install` - i.e. most developers use 4943!

**Pull request merged**